cilium-dbg/cmd/bpf_frag.go

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// BPFFragCmd represents the bpf command
+var BPFFragCmd = &cobra.Command{
+	Use:   "frag",
```

Comment:
nit: we could add a alias for "fragments" to make this a bit easier to use.

Reply:
```go
Use: "frag",
Aliases: []string{"fragments"}
```
This?

Reply:
Yes :+1: